usage: conda [-h] [--no-plugins] [-V] COMMAND ...
conda: error: argument COMMAND: invalid choice: 'deactivate' (choose from 'clean', 'compare', 'config', 'create', 'info', 'init', 'install', 'list', 'notices', 'package', 'remove', 'uninstall', 'rename', 'run', 'search', 'update', 'upgrade', 'build', 'content-trust', 'convert', 'debug', 'develop', 'doctor', 'index', 'inspect', 'metapackage', 'render', 'skeleton', 'token', 'repo', 'pack', 'env', 'verify', 'server')
usage: conda [-h] [--no-plugins] [-V] COMMAND ...
conda: error: argument COMMAND: invalid choice: 'deactivate' (choose from 'clean', 'compare', 'config', 'create', 'info', 'init', 'install', 'list', 'notices', 'package', 'remove', 'uninstall', 'rename', 'run', 'search', 'update', 'upgrade', 'build', 'content-trust', 'convert', 'debug', 'develop', 'doctor', 'index', 'inspect', 'metapackage', 'render', 'skeleton', 'token', 'env', 'repo', 'verify', 'pack', 'server')
no change     /curc/sw/anaconda3/2023.09/condabin/conda
no change     /curc/sw/anaconda3/2023.09/bin/conda
no change     /curc/sw/anaconda3/2023.09/bin/conda-env
no change     /curc/sw/anaconda3/2023.09/bin/activate
no change     /curc/sw/anaconda3/2023.09/bin/deactivate
no change     /curc/sw/anaconda3/2023.09/etc/profile.d/conda.sh
no change     /curc/sw/anaconda3/2023.09/etc/fish/conf.d/conda.fish
no change     /curc/sw/anaconda3/2023.09/shell/condabin/Conda.psm1
no change     /curc/sw/anaconda3/2023.09/shell/condabin/conda-hook.ps1
no change     /curc/sw/anaconda3/2023.09/lib/python3.11/site-packages/xontrib/conda.xsh
no change     /curc/sw/anaconda3/2023.09/etc/profile.d/conda.csh
no change     /home/mlippincott@xsede.org/.bashrc
No action taken.
Processing well_fov C4-2 for patient NF0016
Processing directory  of Segmenting Nuclei
  0%|          | 0/12 [00:00<?, ?it/s]  8%|▊         | 1/12 [00:01<00:20,  1.90s/it] 17%|█▋        | 2/12 [00:02<00:11,  1.14s/it] 25%|██▌       | 3/12 [00:03<00:08,  1.11it/s] 33%|███▎      | 4/12 [00:03<00:06,  1.27it/s] 42%|████▏     | 5/12 [00:04<00:05,  1.38it/s] 50%|█████     | 6/12 [00:04<00:04,  1.46it/s] 58%|█████▊    | 7/12 [00:05<00:03,  1.52it/s] 67%|██████▋   | 8/12 [00:06<00:02,  1.57it/s] 75%|███████▌  | 9/12 [00:06<00:01,  1.59it/s] 83%|████████▎ | 10/12 [00:07<00:01,  1.60it/s] 92%|█████████▏| 11/12 [00:08<00:00,  1.59it/s]100%|██████████| 12/12 [00:08<00:00,  1.59it/s]100%|██████████| 12/12 [00:08<00:00,  1.39it/s]
Completed Nuclei Segmentation
Segmenting Cells
Completed Cell Segmentation
Segmenting Organoids
  0%|          | 0/25 [00:00<?, ?it/s]  8%|▊         | 2/25 [00:00<00:02, 10.72it/s] 16%|█▌        | 4/25 [00:00<00:02,  9.87it/s] 20%|██        | 5/25 [00:00<00:02,  9.69it/s] 28%|██▊       | 7/25 [00:00<00:01, 10.34it/s] 36%|███▌      | 9/25 [00:00<00:01,  9.88it/s] 40%|████      | 10/25 [00:01<00:01,  9.74it/s] 48%|████▊     | 12/25 [00:01<00:01, 10.26it/s] 56%|█████▌    | 14/25 [00:01<00:01,  9.87it/s] 60%|██████    | 15/25 [00:01<00:01,  9.76it/s] 68%|██████▊   | 17/25 [00:01<00:00, 10.23it/s] 76%|███████▌  | 19/25 [00:01<00:00,  9.82it/s] 80%|████████  | 20/25 [00:02<00:00,  9.68it/s] 88%|████████▊ | 22/25 [00:02<00:00, 10.16it/s] 96%|█████████▌| 24/25 [00:02<00:00,  9.78it/s]100%|██████████| 25/25 [00:02<00:00,  9.67it/s]100%|██████████| 25/25 [00:02<00:00,  9.89it/s]
  0%|          | 0/10 [00:00<?, ?it/s] 10%|█         | 1/10 [00:02<00:26,  2.94s/it] 20%|██        | 2/10 [00:05<00:20,  2.55s/it] 30%|███       | 3/10 [00:08<00:20,  2.87s/it] 40%|████      | 4/10 [00:09<00:12,  2.13s/it] 50%|█████     | 5/10 [00:10<00:08,  1.72s/it] 60%|██████    | 6/10 [00:11<00:06,  1.53s/it] 70%|███████   | 7/10 [00:12<00:04,  1.36s/it] 80%|████████  | 8/10 [00:17<00:04,  2.35s/it] 90%|█████████ | 9/10 [00:18<00:02,  2.01s/it]100%|██████████| 10/10 [00:19<00:00,  1.70s/it]100%|██████████| 10/10 [00:19<00:00,  1.94s/it]
Completed Organoid Segmentation
Decoupling nuclei
0it [00:00, ?it/s]2it [00:00, 19.76it/s]4it [00:00, 16.31it/s]6it [00:00, 15.27it/s]8it [00:00, 14.97it/s]10it [00:00, 14.81it/s]12it [00:00, 10.98it/s]13it [00:00, 13.19it/s]
Traceback (most recent call last):
  File "/projects/mlippincott@xsede.org/software/anaconda/envs/GFF_segmentation/lib/python3.11/site-packages/pandas/core/indexes/base.py", line 3805, in get_loc
    return self._engine.get_loc(casted_key)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "index.pyx", line 167, in pandas._libs.index.IndexEngine.get_loc
  File "index.pyx", line 196, in pandas._libs.index.IndexEngine.get_loc
  File "pandas/_libs/hashtable_class_helper.pxi", line 7081, in pandas._libs.hashtable.PyObjectHashTable.get_item
  File "pandas/_libs/hashtable_class_helper.pxi", line 7089, in pandas._libs.hashtable.PyObjectHashTable.get_item
KeyError: 'label'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/gpfs/alpine1/scratch/mlippincott@xsede.org/GFF_3D_organoid_profiling_pipeline/2.segment_images/scripts/4.reconstruct_3D_masks.py", line 372, in <module>
    mask[mask == tmp_df.iloc[i]["original_label"]] = tmp_df.iloc[i]["label"]
                                                     ~~~~~~~~~~~~~~^^^^^^^^^
  File "/projects/mlippincott@xsede.org/software/anaconda/envs/GFF_segmentation/lib/python3.11/site-packages/pandas/core/series.py", line 1121, in __getitem__
    return self._get_value(key)
           ^^^^^^^^^^^^^^^^^^^^
  File "/projects/mlippincott@xsede.org/software/anaconda/envs/GFF_segmentation/lib/python3.11/site-packages/pandas/core/series.py", line 1237, in _get_value
    loc = self.index.get_loc(label)
          ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/projects/mlippincott@xsede.org/software/anaconda/envs/GFF_segmentation/lib/python3.11/site-packages/pandas/core/indexes/base.py", line 3812, in get_loc
    raise KeyError(key) from err
KeyError: 'label'
Decoupling cell
0it [00:00, ?it/s]2it [00:00, 10.35it/s]4it [00:00,  8.05it/s]5it [00:00,  7.75it/s]6it [00:00,  7.57it/s]7it [00:00,  7.40it/s]8it [00:01,  7.30it/s]9it [00:01,  7.22it/s]10it [00:01,  7.09it/s]11it [00:01,  7.02it/s]13it [00:01,  9.10it/s]13it [00:01,  8.01it/s]
Decoupling organoid
0it [00:00, ?it/s]2it [00:00, 19.79it/s]4it [00:00, 10.72it/s]6it [00:00,  8.57it/s]8it [00:01,  6.51it/s]9it [00:01,  6.01it/s]10it [00:01,  5.62it/s]11it [00:01,  5.74it/s]13it [00:01,  8.18it/s]13it [00:01,  7.44it/s]
Traceback (most recent call last):
  File "/gpfs/alpine1/scratch/mlippincott@xsede.org/GFF_3D_organoid_profiling_pipeline/2.segment_images/scripts/4.reconstruct_3D_masks.py", line 275, in <module>
    df["indexes"] = df["index1"].astype(str) + "-" + df["index2"].astype(str)
                    ~~^^^^^^^^^^
  File "/projects/mlippincott@xsede.org/software/anaconda/envs/GFF_segmentation/lib/python3.11/site-packages/pandas/core/frame.py", line 4102, in __getitem__
    indexer = self.columns.get_loc(key)
              ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/projects/mlippincott@xsede.org/software/anaconda/envs/GFF_segmentation/lib/python3.11/site-packages/pandas/core/indexes/range.py", line 417, in get_loc
    raise KeyError(key)
KeyError: 'index1'
Traceback (most recent call last):
  File "/gpfs/alpine1/scratch/mlippincott@xsede.org/GFF_3D_organoid_profiling_pipeline/2.segment_images/scripts/5.create_cytoplasm_masks.py", line 74, in <module>
    ).resolve(strict=True)
      ^^^^^^^^^^^^^^^^^^^^
  File "/projects/mlippincott@xsede.org/software/anaconda/envs/GFF_segmentation/lib/python3.11/pathlib.py", line 993, in resolve
    s = os.path.realpath(self, strict=strict)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen posixpath>", line 416, in realpath
  File "<frozen posixpath>", line 451, in _joinrealpath
FileNotFoundError: [Errno 2] No such file or directory: '/gpfs/alpine1/scratch/mlippincott@xsede.org/GFF_3D_organoid_profiling_pipeline/data/NF0016/processed_data/C4-2/nuclei_masks_reconstructed.tiff'
Segmentation complete
